modules/cmd.js

```diff
@@ -80,26 +81,72 @@ export const cmdToPromise = (cmd, dispatch, getState) => {
             }
             else resolve(result)
           })
-        }).then(flatten)     }
+        }).then(flatten)
+      }
       else return null
 
     case cmdTypes.MAP:
       const possiblePromise = cmdToPromise(cmd.nestedCmd, dispatch, getState)
       if (!possiblePromise) return null
       return possiblePromise.then((actions) => actions.map(cmd.tagger))
 
+    /*case cmdTypes.CALLBACK:
```

Comment:
it could be useful in the future if we want to support callback functions. not sure yet. i figure it can be deleted later.